Sources/Stripe/Models/Invoices/Invoice.swift

```diff
@@ -0,0 +1,162 @@
+        self.isForgiven = try node.get("forgiven")
+        self.isLiveMode = try node.get("livemode")
+        self.isPaid = try node.get("paid")
+        self.nextPaymentAttempt = try node.get("next_payment_attempt")
+        self.periodStart = try node.get("period_start")
+        self.periodEnd = try node.get("period_end")
+        self.receiptNumber = try node.get("receipt_number")
+        self.startingBalance = try node.get("starting_balance")
+        self.statementDescriptor = try node.get("statement_descriptor")
+        self.subscription = try node.get("subscription")
+        self.subtotal = try node.get("subtotal")
+        self.total = try node.get("total")
+        self.tax = try node.get("tax")
+        self.taxPercent = try node.get("tax_percent")
+        
+        self.webhooksDeliveredAt = try node.get("webhooks_delivered_at")
+        
+        self.metadata = try node.get("metadata")
+        
+        if let currency = node["currency"]?.string {
+            self.currency = StripeCurrency(rawValue: currency)
+        }
+        
+        if let lines = node["lines"]?.object, let data = lines["data"]?.array {
+            self.lines = try data.map({ try InvoiceItem(node: $0) })
+        }
+        
+    }
+    
+    public func makeNode(in context: Context?) throws -> Node {
+        let object: [String: Any?] = [
+            "id": self.id,
+            "object": self.object,
+            "amount_due": self.amountDue,
+            "application_fee": self.applicationFee,
+            "attempt_count": self.attemptCount,
+            "attempted": self.hasAttemptedCharge,
+            "charge": self.charge,
+            "closed": self.isClosed,
+            "customer": self.customer,
+            "date": self.date,
+            "description": self.description,
+            "discount": self.discount,
+            "ending_balance": self.endingBalance,
+            "forgiven": self.isForgiven,
+            "livemode": self.isLiveMode,
+            "paid": self.isPaid,
+            "next_payment_attempt": self.nextPaymentAttempt,
+            "period_start": self.periodStart,
+            "period_end": self.periodEnd,
+            "receipt_number": self.receiptNumber,
+            "starting_balance": self.startingBalance,
+            "statement_descriptor": self.statementDescriptor,
+            "subscription": self.subscription,
+            "subtotal": self.subtotal,
+            "total": self.total,
+            "tax": self.tax,
+            "tax_percent": self.taxPercent,
+            "webhooks_delivered_at": self.webhooksDeliveredAt,
+            
+            "metadata": self.metadata,
+            
+            "currency": self.currency?.rawValue
+        ]
+        
+        return try Node(node: object)
+    }
+}
+
+public final class InvoiceList: StripeModelProtocol {
+    
+    public private(set) var object: String?
+    public private(set) var hasMore: Bool?
+    public private(set) var items: [Invoice]?
+    
```

Comment:
Any reason why we're missing the URL that's passed back?

Reply:
After looking at it, I don't think we need it. I feel like it can be removed across the board too.

Reply:
Well I really only added them because I thought someone somewhere might need it at some point and I wanted it to be as complete as possible.

Reply:
Yeah, thats what I had figured to begin with as well, but since it's not the full FQDN and API Endpoint, nor do we allow raw URL queries, I don't think its needed. For consistency for now, I can add?

Reply:
Yea it's one of those thing where I don't use or really care about a URL lol. It's not really that important 🌚